apps/server/src/modules/task-card/controller/mapper/task-card.mapper.ts

```diff
@@ -27,6 +26,7 @@ export class TaskCardMapper {
 			task: taskResponse,
```

Comment:
the dto value for `cardElements` above should take its optional nature into consideration like with `course` below